docs/references/javascript/organization/members.mdx

```diff
@@ -10,41 +10,80 @@ These are all methods on [the Organization class](/docs/references/javascript/or
 ## `getMemberships()`
 
 ```typescript
-function getMemberships(params?: GetMembershipsParams): Promise<OrganizationMembership[]>;
+// Updated API
+function getMemberships(params?: GetMembershipsParams): Promise<PaginatedResponse<OrganizationMembership>>;
+
+// Deprecated API
+function getMemberships(params?: DeprecatedGetMembershipsParams): Promise<OrganizationMembership[]>;
 ```
 
 Retrieve the members of the selected
```

Comment:
Not sure why this sentence was never finished - can we update it? I'm not sure what it's supposed to say, but I tried to add something here:

```suggestion
Retrieve the members of the currently active organization.
```